packages/clay-drop-down/src/DropDownWithItems.tsx

```diff
@@ -360,8 +360,9 @@ const RadioGroup: React.FunctionComponent<IItem & IInternalItem> = ({
 	name,
 	onChange = () => {},
 	spritemap,
+	value: initialValue = ''
```

Comment:
@victorg1991 I think here this value could be called `defaultValue` just to avoid some confusion, what do you think?

Reply:
Done :)